validator_client/beacon_node_fallback/src/beacon_head_monitor.rs

```diff
@@ -0,0 +1,148 @@
+use crate::BeaconNodeFallback;
+use eth2::types::EthSpec;
```

Comment:
We should import from the types crate directly for this import:
```rust
use types::EthSpec;
``` 
The `eth2` re-export of `types` will get removed eventually so this would save a conflict later